### ISSUE TITLE
Error list: path of files not read out by screen readers

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51998]
**Environment Details:** 
VSCode Version : 1.24.0
OS Version : Win10
Mac Version: HighSierra10.13.5

**Additional Details:**
ScreenReader Version: NVDA2018.2
MAS violated: MAS 1.3.1

Repro Steps:
1. Launch VS Code while NVDA is ON.
2. Go to View menu and Select Problems.
3. Tab till any of the opened snippets file in the Problems tab.

Actual:
NVDA does not read the path written beside the snippets file like (C://Users/v-chranj/appdata/Roaming).
Issue has also been observed with Voiceover.

Expected:
NVDA should read each and every content that is visible on the screen.

Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

User Impact:
User will not know the path where these files are getting saved.

MAS Reference:
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Attachment for Reference: 
![problems](https://user-images.githubusercontent.com/39204909/41894912-2a677796-793e-11e8-87c8-3d609ac93ce3.JPG)

[A11y VSCode Problems NVDA notreadingpath.pptx](https://github.com/Microsoft/vscode/files/2105530/A11y.VSCode.Problems.NVDA.notreadingpath.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by sandy081** [comment]
@AccessibilityTestingTeam-TCS Can you please let me know what do you mean by the path written beside the snippets file? 

Also do not understand the user impact you mentioned.

Can you please provide the screenshot of the feature you mentioned?

**Comment by AccessibilityTestingTeam-TCS** [comment]
Attached the screenshot. On mousehover, NVDA is reading the path but ideally it should also read through screenreader commands.

**Comment by isidorn** [comment]
Path is not read out for files in the problem view. This feels like an easy fix, thus assigning to July.
Aria-label of the item should contain the path.
